music21/musicxml/m21ToXml.py

```diff
@@ -3863,6 +3887,49 @@ def pitchToXml(self, p: pitch.Pitch):
         _setTagTextFromAttribute(p, mxPitch, 'octave', 'implicitOctave')
         return mxPitch
 
+    def unpitchedToXml(self,
+                       up: note.Unpitched,
+                       noteIndexInChord: int = 0,
+                       chordParent: chord.ChordBase = None) -> Element:
+        '''
+        Convert a :class:`~music21.note.Unpitched` to a <note>
+        with an <unpitched> subelement.
+
+        >>> up = note.Unpitched('D5')
```

Comment:
for constructing Unpitched objects, can we specify `displayName='D5'`  -- I think that otherwise it might be an issue later where people are surprised that displayName='C4' appears one line below the staff no matter what the clef is.

Reply:
I didn't quite follow this.

Reply:
Watch the next diff to see what I meant.

Reply:
Just let's never create:

```
note.Unpitched('D5')
```

always instead

```
note.Unpitched(displayName='D5')
```

because even if the current clef is Bass clef, displayName refers to the location as if the staff were in treble clef no matter what.